src/api/server/configuration.rst

```diff
@@ -16,316 +16,14 @@
 Configuration
 =============
 
-The CouchDB Server Configuration API provide an interface to query and update
-the various configuration values within a running CouchDB instance.
+The classical CouchDB Server Configuration API located at ``/_config`` will
+return a 404 in CouchDB 2.0 and got replaced by :ref:`HTTP API <api/node>`.
 
-``/_config``
```

Comment:
well, this opens a point of discussion, how do document the difference between the cluster and the node port? Do we at all?

IMHO, the focus should be on documenting the cluster port, as that’s the main public API.

We should document the node-specific API too, but I think it should be separate, or specifically marked, that it is node-port only. I’ll leave it to the people who are better at designing an information architecture for our docs, to decide what is better, my gut feeling says “separate sections”, but it can be convinced otherwise :)